cpu/cc2538/Makefile.include

```diff
@@ -1,3 +1,7 @@
 export CPU_ARCH := cortex-m3
-
+ROM_LEN ?= 0x80000 
```

Comment:
Are the ROM_LEN allways the same? can it change depending on the cpu version?

Reply:
![CC2538XXXX](https://user-images.githubusercontent.com/25896872/59767454-a8735a00-92a2-11e9-99fb-f7fab6d7b198.png)

Reply:
I'm not sure whether if I need to decrease the ROM_LEN to account for the CCA. However when I tried 0x7FFD4 (which is 512K - 44), the flashing failed
> Writing 260116 bytes starting at address 0x002407EA
> Target returned: 0x43, Invalid address

Reply:
Ok so since it has different length we need to set ROM_LEN and RAM_LEN according to CPU_MODEL so its taking into account for all options since this is in a common cc2538 file.

> I'm not sure whether if I need to decrease the ROM_LEN to account for the CCA. 

 I don't think so. The problem right now is that the entry point is in the CCA field so we NEED to write it every time for normal applications, but make it always be the bootloader for riotboot.

> However when I tried 0x7FFD4 (which is 512K - 44), the flashing failed

How did it fail?

Reply:
The flashing failed:
> Writing 260116 bytes starting at address 0x002407EA
Target returned: 0x43, Invalid address

Reply:
when i tried 0x7FF00, it flashed correctly